app/controllers/search_controller.rb

```diff
@@ -5,6 +5,7 @@ def index
     permitted = params.require(:search).permit(:email)
     raise ActionController::ParameterMissing if permitted.blank?
     @matching_users = search_by_email(permitted[:email].strip)
+    @email_query = permitted[:email]
```

Comment:
We should also test this in `spec/controllers/search_controller_spec.rb`!

Reply:
k!